azure/durable_functions/models/DurableOrchestrationClient.py

```diff
@@ -73,8 +73,17 @@ async def start_new(self,
 
         if response[0] <= 202 and response[1]:
             return response[1]["id"]
+        elif response[0] == 400:
+            # Orchestrator not found
+            exception_data = response[1]
+            exception_message = exception_data["ExceptionMessage"]
+            stack_trace = exception_data["StackTrace"]
+            exception_message += '\n' + stack_trace
```

Comment:
Why bother including a stack-trace for this specific case? That's for .NET extension code and I don't expect it to be relevant to a Python developer (in fact, it might make them feel uncomfortable). Can we report just the "ExceptionMessage" field?

Reply:
When you said:
"_If you can detect that the response is JSON and has a "Message" field, then the error could simply be the Message text rather than "The operation failed with an unexpected status code..." followed by the full payload that includes this ugly call-stack._"
I thought you meant that you wanted me to surface the stack trace as well 😕 , maybe that's why we've been having this back and forth. Yes, we can just throw the exception-message. On it

Reply:
Argh - yes, my comment was confusing. That's my bad. :(